File: generated-sources/kotlin-spring/mojang-authentication/src/main/kotlin/com/github/asyncmc/mojang/authentication/kotlin/spring/model/GameProfile.kt
package com.github.asyncmc.mojang.authentication.kotlin.spring.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param agent 
 * @param id 
 * @param name 
 * @param userId 
 * @param createdAt Unix timestamp in milliseconds
 * @param legacyProfile 
 * @param suspended 
 * @param paid 
 * @param migrated 
 * @param legacy 
 */
data class GameProfile (

        @JsonProperty("agent") val agent: String? = null,

        @JsonProperty("id") val id: String? = null,

        @JsonProperty("name") val name: String? = null,

        @JsonProperty("userId") val userId: String? = null,

        @JsonProperty("createdAt") val createdAt: Long? = null,

        @JsonProperty("legacyProfile") val legacyProfile: Boolean? = null,

        @JsonProperty("suspended") val suspended: Boolean? = null,

        @JsonProperty("paid") val paid: Boolean? = null,

        @JsonProperty("migrated") val migrated: Boolean? = null,

        @JsonProperty("legacy") val legacy: Boolean? = null
) {

}

File: generated-sources/php-ze-ph/mojang-authentication/src/App/com/github/asyncmc/mojang/authentication/php/ze/ph/model/GameProfile.php
<?php

namespace com.github.asyncmc.mojang.authentication.php.ze.ph.server\com.github.asyncmc.mojang.authentication.php.ze.ph.model;

use Articus\DataTransfer\Annotation as DTA;

/**
 */
class GameProfile
{
    /**
     * @DTA\Data(field="agent", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $agent;
    /**
     * @DTA\Data(field="id", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $id;
    /**
     * @DTA\Data(field="name", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $name;
    /**
     * @DTA\Data(field="userId", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $user_id;
    /**
     * Unix timestamp in milliseconds
     * @DTA\Data(field="createdAt", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $created_at;
    /**
     * @DTA\Data(field="legacyProfile", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"bool"})
     * @var bool
     */
    public $legacy_profile;
    /**
     * @DTA\Data(field="suspended", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"bool"})
     * @var bool
     */
    public $suspended;
    /**
     * @DTA\Data(field="paid", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"bool"})
     * @var bool
     */
    public $paid;
    /**
     * @DTA\Data(field="migrated", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"bool"})
     * @var bool
     */
    public $migrated;
    /**
     * @DTA\Data(field="legacy", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"bool"})
     * @var bool
     */
    public $legacy;
}
